### DISCUSSION TITLE
BossbarMsg placeholder

### DISCUSSION THREAD

**Discussion by MrHarebug** [Zrips/CMI#4011] [Feature Requests]
Is possible use as name player name placeholder?
I mean /cmi bossbarmsg all -sec:20 -n:[playerName] &cHello
(Player name is same as Bossbar name)

Also use placeholder from your Y (Height) to -sec:%player_y% (for example)

**Comment by mrfloris** [comment]
You can learn about CMI placeholders here:
https://www.zrips.net/cmi/placeholders/

And about CMI bossbarmsg here:
https://www.zrips.net/cmi/commands/bossbarmsg-handling/

`cmi bossbarmsg [playerName] &2Hey you! -cmd:"msg [playerName] hey;;cmi heal [playerName]"`

You can use "all" and then use %cmi_user_name% for example, this worked for me. 
<img width="1392" alt="Screenshot 2020-08-19 at 01 22 42" src="https://user-images.githubusercontent.com/28841349/90574960-91880a80-e1ba-11ea-88a1-dd1f3b68f33f.png">
<img width="1392" alt="Screenshot 2020-08-19 at 01 22 19" src="https://user-images.githubusercontent.com/28841349/90574966-96e55500-e1ba-11ea-920d-151ae404243d.png">

**Comment by MrHarebug** [comment]
Yes this is okay but I try use placeholder for playername as bossbar name.
/cmi bossbarmsg all -sec:20 -n:%cmi_user_name% Test
(-n: is bossbar name)
